Exercise1/myFile-rt/src/com/tridiumuniversity/myFile/BTestFile.java

```diff
@@ -0,0 +1,213 @@
+package com.tridiumuniversity.myFile;
+
+import javax.baja.nre.annotations.NiagaraAction;
+import javax.baja.nre.annotations.NiagaraProperty;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.sys.*;
+
+@NiagaraType
+@NiagaraProperty(
+        name = "filename",
```

Comment:
Not a big deal, but per the Exercise instructions, this should technically be `fileName`, not `filename`.